fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSites.kt

```diff
@@ -227,17 +229,47 @@ private fun TopSiteFaviconCard(topSite: TopSite) {
         Box(contentAlignment = Alignment.Center) {
             Surface(
                 modifier = Modifier.size(36.dp),
+                color = FirefoxTheme.colors.layer2,
                 shape = RoundedCornerShape(4.dp),
             ) {
-                Favicon(
-                    url = topSite.url,
-                    size = 36.dp,
-                )
+                when (topSite.url) {
```

Comment:
Since the when checks are already there below, maybe simpler way would be

```
val drawableForUrl = getDrawableForUrl(topSite.url)
                if (drawableForUrl != null) {
                ...
```